lib/sqlalchemy/dialects/postgresql/base.py

```diff
@@ -3402,6 +3402,7 @@ def get_indexes(self, connection, table_name, schema, **kw):
                   ix.indisunique, ix.indexprs, ix.indpred,
```

Comment:
**mike bayer** (zzzeek) wrote:

we are now looking at ix.indpred in the pg_get_expr() function below, so I *think* we can remove ix.indpred here....

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2142